### PR TITLE
Migrate balrog.submitter.api to balrogclient

### PR DESCRIPTION
**Pull request by srfraser**
Moved from hg.m.o/build/tools to github
Basic test added
Python 3.5 compatibility

Minimal refactoring for the moment, to reduce the number of things to track during the module's location change.

### CODE REVIEW

File: python-lib/tox.ini [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+[base]
+deps =
+    nose==1.3.0
+    rednose==0.4.1
+    coverage==3.6
+    pep8==1.4.3
+    jinja2==2.6
+    mock==1.0.1
+    requests
+    requests_mock
```
Comment (mozbhearsum) [inline python-lib/tox.ini]: Why no pinning on these?

File: python-lib/tox.ini [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+[testenv]
+
+commands =
+    coverage erase
+    coverage run --branch --source {toxinidir}/balrogclient {envbindir}/nosetests -v --with-xunit --rednose --force-color {toxinidir}/balrogclient {posargs}
+
+
+[testenv:py27]
+basepython = python2.7
```
Comment (mozbhearsum) [inline python-lib/tox.ini]: Yay, 2.7 and 3.5! We may want to run against 2.6 as well, if we still use that in production anywhere...I'm trying to find out if that's the case or not.

File: python-lib/setup.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/env python
+
+from setuptools import setup, find_packages
+
+setup(
+    name="balrogclient",
+    version="0.0.1",
+    description="Mozilla RelEng Toolkit",
```
Comment (mozbhearsum) [inline python-lib/setup.py]: This should probably be "Balrog Admin API Client" or something like that.

File: python-lib/tox.ini (hunk 1 above)
Comment (srfraser) [inline python-lib/tox.ini]: Fixed, update coming.

File: python-lib/setup.py (hunk 3 above)
Comment (srfraser) [inline python-lib/setup.py]: Fixed, update coming.

**Pull request merged**